@navikt/core/react/src/layout/box/Box.stories.tsx

```diff
@@ -0,0 +1,318 @@
+import React, { ReactNode } from "react";
+import type { Meta } from "@storybook/react";
+import { BodyLong, Detail, Heading } from "../../typography";
+import { Box } from "./Box";
+import { HGrid, HStack, VStack } from "../..";
+import { BackgroundSpecifier, BorderRadiusSpecifier } from "./types";
+import { ChevronRightIcon } from "@navikt/aksel-icons";
+
+export default {
+  title: "ds-react/Box",
```

Comment:
Burde alle stories på primitives grupperes?
```suggestion
  title: "ds-react/Primitives/Box",
```

Reply:
enig! :)